contents/staticAnalysis/FindBugs.md

```diff
@@ -0,0 +1,59 @@
+# FindBugs
+
+Author: [Xiao Pu](https://nus-oss.github.io/cs3281-website/students/AY1617S2/xiaoPu/xiaoPu-Resume.html)
+
+## Overview
+
+FindBugs is a static analysis to find "bugs" in **Java** programme. It looks for "bug patterns" in code and signals possible violations.
```

Comment:
_Could add some words to make it clearer:_
...is a static analysis **tool for us** to find "bugs" in **Java programs** .